src/components/ContentSidebar/ActivityFeed/version/CollapsedVersion.js

```diff
@@ -52,7 +52,7 @@ function getMessageForAction(
                 numberOfCollaborators,
                 versions: (
                     <span className='bcs-version-range'>
-                        {versionStart} - {versionEnd}
+                        {version_start} - {version_end}
```

Comment:
Looks like this section and the above are duplicated, can be made into a variable

```jsx
const versionRange: React.Node = (
    <span className='bcs-version-range'>
        {version_start} - {version_end}
    </span>
);
```

Reply:
👍